src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/PureSphericalHarmonic.cpp

```diff
@@ -15,24 +16,25 @@
 
 namespace CurvedScalarWave::AnalyticData {
 
-PureSphericalHarmonic::PureSphericalHarmonic(
-    double radius, const double width, const size_t l, const int m,
-    const Options::Context& /*context*/)
-    : radius_(radius), width_sq_(width * width), l_mode_(l), m_mode_(m) {
-  // LCOV_EXCL_START
-  if (abs(m_mode_) > static_cast<int>(l_mode_)) {
-    ERROR(
+PureSphericalHarmonic::PureSphericalHarmonic(const double radius,
+                                             const double width,
+                                             std::pair<size_t, int> mode,
+                                             const Options::Context& context)
+    : radius_(radius), width_sq_(width * width), mode_{std::move(mode)} {
+  if (abs(mode.second) > static_cast<int>(mode.first)) {
```

Comment:
(not requesting this change) Sometimes it can be nice to use structured bindings to name components, like `const auto& [l_mode, m_mode] = mode;`